### PR TITLE
Sh modal

### PR DESCRIPTION
**Pull request by sarahholder**
This is a work in progress -- NOT finished. Modal shows on click event and populates name from selectedAirport.

### CODE REVIEW

**Comment by sarahholder** [comment]
Ready for review

File: src/javascripts/components/hub/editAirport.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+          </div>
+        </div>
+        <div class="form-row">
+          <div class="col-md-2 mb-3">
+            <label for="edit-code">Airport Code: </label>
+            <input type="text" class="form-control" id="edit-code" placeholder="${airport.airportCode}" value="${airport.airportCode}">
+          </div>
+          <div class="col-md-2 mb-3">
+            <label for="edit-flights">Flights: </label>
+            <input type="text" class="form-control" id="edit-flights" placeholder="${airport.numFlights}" value="${airport.numFlights}">
```
Comment (zoeames) [inline src/javascripts/components/hub/editAirport.js:44]: should be: type="number"

File: src/javascripts/components/hub/editAirport.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+          </div>
+          <div class="col-md-2 mb-3">
+          <label for="edit-restaurants">Restaurants: </label>
+          <input type="text" class="form-control" id="edit-restaurants" placeholder="${airport.numRestaurants}" value="${airport.numRestaurants}">
```
Comment (zoeames) [inline src/javascripts/components/hub/editAirport.js:48]: should be: type="number"

File: src/javascripts/components/hub/editAirport.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+          </div>
+          <div class="col-md-2 mb-3">
+            <label for="edit-shuttles">Shuttles: </label>
+            <input type="text" class="form-control" id="edit-shuttles" placeholder="${airport.numShuttles}" value="${airport.numShuttles}">
```
Comment (zoeames) [inline src/javascripts/components/hub/editAirport.js:52]: should be: type="number"

File: src/javascripts/components/hub/editAirport.js [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+          </div>
+          <div class="col-md-2 mb-3">
+            <label for="edit-hotels">Hotels: </label>
+            <input type="text" class="form-control" id="edit-hotels" placeholder="${airport.numHotels}" value="${airport.numHotels}">
```
Comment (zoeames) [inline src/javascripts/components/hub/editAirport.js:56]: should be: type="number"

File: src/javascripts/components/hub/hub.js [hunk 5]
```diff
@@ -12,10 +16,138 @@ const removeAirport = (e) => {
...
+  e.preventDefault();
+  const radio = $('#radio').is(':checked');
+  const newAirport = {
+    airportName: $('#name').val(),
+    imgUrl: $('#image').val(),
+    city: $('#city').val(),
+    state: $('#state').val(),
+    country: $('#country').val(),
+    airportCode: $('#airportCode').val(),
+    numFlights: $('#numFlights').val(),
```
Comment (zoeames) [inline src/javascripts/components/hub/hub.js:29]: all the things that start with 'num' should save a numbers - you will need to multiply each one by the number 1

File: src/javascripts/components/hub/hub.js [hunk 6]
```diff
@@ -12,10 +16,138 @@ const removeAirport = (e) => {
...
+  const radio = $('#radio').is(':checked');
+  const selectedAirportId = e.target.closest('.get-edit-id').id;
+  const changedAirport = {
+    airportName: $('#edit-name').val(),
+    imgUrl: $('#edit-imgUrl').val(),
+    city: $('#edit-city').val(),
+    state: $('#edit-state').val(),
+    country: $('#edit-country').val(),
+    airportCode: $('#edit-code').val(),
+    numFlights: $('#edit-flights').val(),
```
Comment (zoeames) [inline src/javascripts/components/hub/hub.js:56]: same as above make sure these are numbers

**Comment by zoeames** [comment]
A few more things: 
1.  you need to add description to your seed data and to your forms - there is lorem ipsum on each card that isn't changeable
2.  when i have isInternation = true and open the edit form domestic is checked
3.  missed this last time - in your seed data all the numStuff values need to be numbers - they are currently strings like "25".  remove the quotes and re-seed
4. when someone saves or edits those number fields should save to firebase as numbers

**Pull request closed without merging**